src/views/ScoreCalc.vue

```diff
@@ -377,109 +530,137 @@ export default {
+          noteType === "break"
+            ? this.breakBonusScore *
+              this.bonusJudgementPercentage[judgementType]
+            : 0
       );
     },
     playerScoreSubtotalsDX() {
       return this.calcPlayerNoteScoreSubtotals(
         (noteType, judgementType) =>
-          this.noteBaseScores[noteType] * this.baseJudgementsPercentage[judgementType],
+          this.noteBaseScores[noteType] *
+          this.baseJudgementsPercentage[judgementType],
         (noteType, judgementType) =>
-          noteType === 'break' ? this.breakBonusScoreDX * this.bonusJudgementPercentageDX[judgementType] : 0,
+          noteType === "break"
+            ? this.breakBonusScoreDX *
+              this.bonusJudgementPercentageDX[judgementType]
+            : 0
       );
     },
     playerDeluxeScoreSubtotals() {
       return this.calcPlayerNoteScoreSubtotals(
-        (noteType, judgementType) => this.baseJudgementsDeluxeScore[judgementType],
-        () => 0,
+        (noteType, judgementType) =>
+          this.baseJudgementsDeluxeScore[judgementType],
+        () => 0
       );
     },
 
     /* Player score */
 
     playerScore() {
-      return this.noteTypes.reduce((acc, noteType) => acc + this.playerScoreSubtotals[noteType], 0);
+      return this.noteTypes.reduce(
+        (acc, noteType) => acc + this.playerScoreSubtotals[noteType],
+        0
+      );
     },
     playerScoreDX() {
-      return this.noteTypes.reduce((acc, noteType) => acc + this.playerScoreSubtotalsDX[noteType], 0);
+      return this.noteTypes.reduce(
+        (acc, noteType) => acc + this.playerScoreSubtotalsDX[noteType],
+        0
+      );
     },
     playerDeluxeScore() {
-      return this.noteTypes.reduce((acc, noteType) => acc + this.playerDeluxeScoreSubtotals[noteType], 0);
+      return this.noteTypes.reduce(
+        (acc, noteType) => acc + this.playerDeluxeScoreSubtotals[noteType],
+        0
+      );
     },
 
     /* Player achievement */
 
     playerAchievement() {
-      return 100.0 * this.playerScore / this.totalBaseScore;
+      return (100.0 * this.playerScore) / this.totalBaseScore;
     },
     playerAchievementDX() {
-      return 100.0 * this.playerScoreDX / this.totalBaseScore;
+      return (100.0 * this.playerScoreDX) / this.totalBaseScore;
     },
     playerDeluxeAchievement() {
-      return 100.0 * this.playerDeluxeScore / this.totalBaseDeluxeScore;
+      return (100.0 * this.playerDeluxeScore) / this.totalBaseDeluxeScore;
     },
 
     /* Player performance */
 
     playerPerformanceOverview() {
       return this.judgementTypes.reduce((acc, judgementType) => {
-        acc[judgementType] = this.noteTypes.map(noteType =>
-          this.playerPerformance[noteType][judgementType]
-        ).reduce((acc, e) => acc + e, 0);
+        acc[judgementType] = this.noteTypes
+          .map(noteType => this.playerPerformance[noteType][judgementType])
+          .reduce((acc, e) => (e ? acc + e : acc), 0);
```

Comment:
changed: fix `NaN` issue

Reply:
There'll be no NaN issue if the counts of all judgement types are provided.